### DISCUSSION TITLE
Welcome to android-signaturepad Discussions!

### DISCUSSION THREAD

**Discussion by warting** [warting/android-signaturepad#18] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ichsanudin98** [comment]
hi, thanks before for your library. i helped with it. But i wanna ask, how i can set signature from my bitmap?

**Reply by warting** [reply]
Hi! I'm not sure I understand the question. Du you want to add a signature to an existing bitmap? Du you have an example?

**Reply by ichsanudin98** [reply]
yes, like that. set signature from existing bitmap

**Reply by warting** [reply]
Sorry, I still don't quite understand the question. If you already have a signature in a bitmap I don't see the usecase for using this library.
